errors/doc.go

```diff
@@ -1,19 +1,19 @@
 /**
- Package errors implements custom error interfaces for weave.
+Package errors implements custom error interfaces for weave.
```

Comment:
???

This is part of the other PR, no?

Reply:
well, that's what happens after go fmt. I will remove this change